integration_tests/support/commands.ts

```diff
@@ -6,13 +6,15 @@ import Page from '../pages/page'
 import FindRecipientByPrisonNumberPage from '../pages/barcode/findRecipientByPrisonNumber'
 
 Cypress.Commands.add('signIn', (options = { failOnStatusCode: true }) => {
+  cy.task('stubGetPrisonRegister')
```

Comment:
Getting the prison register is required by pretty much all of our integrations tests, so I decided it was easiest/cleanest to set it up as part of the `signIn` (DPS sign in), and `signInAsLegalSender`